Testing/functional/tests/cypress/integration/ui/services/view.js

```diff
@@ -0,0 +1,30 @@
+const servicesTestsConstants = require("./servicesTestConstants");
+const { AuthMethod } = require("../../../support/constants");
+
+describe("Authenticated Services View", () => {
+    beforeEach(() => {
+        cy.configureSettings({
+            services: {
+                enabled: true,
+            },
+        });
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak,
+            "/home"
+        );
+        cy.visit(servicesTestsConstants.servicesUrl);
+    });
+
+    afterEach(() => {
+        Cypress.session.clearAllSavedSessions();
+    });
+
+    // Minimum to test skeleton of services page
+    it("View should show title and header", () => {
+        cy.get("h1#subject").should("have.text", "Services");
+        cy.get("h5.my-3").should("be.visible");
```

Comment:
We don't usually check the existence or content of static page elements in the functional tests.

Reply:
This was just for skeleton page's UI test, I went along with the joke in the morning meeting.

There isn't anything to test on the page but to ensure that the requirements of the task were facilitated.

Should I remove this?

Reply:
I'd recommend removing them; the selector for the second one is very brittle, so if we did want to check that element, we would assign it a data-testid.